libr/anal/anal.c

```diff
@@ -419,6 +419,8 @@ R_API int r_anal_purge (RAnal *anal) {
 	sdb_reset (anal->sdb_zigns);
```

Comment:
r_anal_purge should return void and have no space before (. can you fix this in this same PR?

also rebase and cross the fingers to be green

Reply:
crosses fingered